shimming-toolbox/test/cli/test_cli_b0shim.py

```diff
@@ -1622,22 +1613,194 @@ def test_combine_shim_coefs_sl(self):
                 f.write("1,2,3, 4,\n")
                 f.write("5,6,7, 4\n")
```

Comment:
Shouldn't it be counted as the same number of channels?

Reply:
Not anymore, the format needs to be the same now.

Reply:
There is a format with an extra comma and one without?

Reply:
Yes, it's not necessary, it's testing that it does not throw an error based on that. It should be able to parse the input whether there are spaces and commas at the end (or no comma)